aten/src/ATen/native/native_functions.yaml

```diff
@@ -8319,6 +8319,10 @@
   python_module: linalg
   variants: function
 
+- func: linalg_tensorinv(Tensor self, int? ind=2) -> Tensor
```

Comment:
What's the plan for out support?

Reply:
Can you elaborate on how `ind` is parsed? It's both optional and it has a default value of 2?